backend/tests/api/test_devices.py

```
@@ -0,0 +1,34 @@
+        "room": "API Room",
+        "user_id": user.id,
+    }
+    response = await client.post(f"{settings.API_V1_STR}/devices/", json=device_data)
+    assert response.status_code == 200
+    content = response.json()
+    assert content["name"] == device_data["name"]
+    assert content["user_id"] == user.id
+
+
+@pytest.mark.asyncio
+async def test_read_devices_api(client: AsyncClient, db_session: AsyncSession) -> None:
```

Ruff (0.14.7)

Unused function argument: `db_session`

(ARG001)